index.html

```diff
@@ -64,6 +62,231 @@ <h1>I’m Hussen. Glad to see you!</h1>
+          </div>
+        </div>
+        <!-- card works  3-->
+        <div class="work-container">
+          <div class="cards">
+            <div class="snapshot-portofolio">
+              <img src="img/work-img/Snapshoot Portfolio 3.png" alt="SP" />
+            </div>
+            <div class="blocks">
+              <h2 class="project-title txtWrapper">Tonic</h2>
+              <div class="tframes">
+                <h3>CANOPY</h3>
+                <ol class="frames-list">
+                  <li>
+                    <img src="img/work-img/Counter.png" />
+                  </li>
+                  <li>Back End Dev</li>
+                  <li>
+                    <img src="img/work-img/Counter.png" />
+                  </li>
+                  <li>2015</li>
+                </ol>
+              </div>
+              <p class="project-text txtWrapper">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tag-list">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+              <div class="blocks-buttonWrapper">
+                <button class="blocks-button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- card works  4-->
+        <div class="work-container">
+          <div class="cards">
+            <div class="snapshot-portofolio">
+              <img src="img/work-img/Snapshoot Portfolio 4.png" alt="SP" />
+            </div>
+            <div class="blocks">
+              <h2 class="project-title txtWrapper">Multi-Post Stories</h2>
+              <div class="tframes">
+                <h3>CANOPY</h3>
+                <ol class="frames-list">
+                  <li>
+                    <img src="img/work-img/Counter.png" />
+                  </li>
+                  <li>Back End Dev</li>
+                  <li>
+                    <img src="img/work-img/Counter.png" />
+                  </li>
+                  <li>2015</li>
+                </ol>
+              </div>
+              <p class="project-text txtWrapper">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="tag-list">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+              <div class="blocks-buttonWrapper">
+                <button class="blocks-button">See Project</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="about-section">
+        <div class="about-Container">
+          <h2 class="headingTxt">About Myself</h2>
+          <div class="text-2">
+            <p>
+              Hello I’m a software developer! I can help you build a product ,
```

Comment:
- [Optional] Dear @HussenYesufAlli, Your design is great and I like the changes you made as requested. But I notice there is a big gap between your work and the about section. This is a bit different compared with the figma design. Please take a look. 
![image](https://user-images.githubusercontent.com/69077061/227745269-24faae25-ca87-41c0-92ed-5688c4affeb3.png)